src/Microsoft.ML.Data/MLContext.cs

```diff
@@ -82,7 +82,31 @@ public sealed class MLContext : IHostEnvironment
         /// <summary>
         /// Create the ML context.
         /// </summary>
-        /// <param name="seed">Random seed. Set to <c>null</c> for a non-deterministic environment.</param>
+        /// <param name="seed">Seed for MLContext's random number generator. See the remarks for more details.</param>
+        /// <remarks>
+        /// Many operations in ML.NET require randomness, such as
+        /// random data shuffling, random sampling, random parameter initialization,
+        /// random permutation, random feature selection, and many more.
+        /// MLContext's random number generator is the global source of randomness for
+        /// all of such random operations.
+        ///
+        /// If a seed is provided by <paramref name="seed"/>, MLContext environment becomes
```

Comment:
```suggestion
        /// If a fixed seed is provided by <paramref name="seed"/>, MLContext environment becomes
```

Reply:
This is great! 👍 This is an explanation I asked for in this ISSUE: https://github.com/dotnet/machinelearning/issues/3048 

Adding @natke so specific documents at docs.microsoft.com will point to this explanation when first introducing the MLContext where seed is used. 

Thank you! 👍